### PR TITLE
Adding ability to install official released NetObserv operator

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Resolves [OCPQE-13592](https://issues.redhat.com/browse/OCPQE-13592)

### CODE REVIEW

**Comment by nathan-weinberg** [comment]
Successful Runs:
`Official`: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/nweinber-e2e-benchmarking-multibranch-pipeline/job/netobserv-perf-tests/354/
`Internal`: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/nweinber-e2e-benchmarking-multibranch-pipeline/job/netobserv-perf-tests/355/
`OperatorHub`: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/nweinber-e2e-benchmarking-multibranch-pipeline/job/netobserv-perf-tests/357
`Source`: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/nweinber-e2e-benchmarking-multibranch-pipeline/job/netobserv-perf-tests/359/

**Review by Amoghrd — COMMENTED** [review]
loki.querierUrl is not required I think. Its not being used in backend automation

**Review by memodi — CHANGES_REQUESTED** [review]
@nathan-weinberg - LGTM, just one minor nit on file naming, let me know wdyt.

File: scripts/netobserv.sh [hunk 1]
```diff
@@ -2,28 +2,32 @@
...
 SCRIPTS_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
 export DEFAULT_SC=$(oc get storageclass -o=jsonpath='{.items[?(@.metadata.annotations.storageclass\.kubernetes\.io/is-default-class=="true")].metadata.name}')
-if [[ $INSTALLATION_SOURCE == "Downstream" ]]; then
-  echo "Using 'Downstream' as INSTALLATION_SOURCE"
-  NOO_SUBSCRIPTION=$SCRIPTS_DIR/subscriptions/netobserv-downstream-subscription.yaml
-  FLOWCOLLECTOR=$SCRIPTS_DIR/flowcollector/flows_v1alpha1_flowcollector_downstream_lokistack.yaml
+if [[ $INSTALLATION_SOURCE == "Official" ]]; then
+  echo "Using 'Official' as INSTALLATION_SOURCE"
+  NOO_SUBSCRIPTION=$SCRIPTS_DIR/subscriptions/netobserv-official-subscription.yaml
+  FLOWCOLLECTOR=$SCRIPTS_DIR/flowcollector/flows_v1alpha1_flowcollector_released_lokistack.yaml
```
[the remark this answers is not in the document]
Comment (nathan-weinberg) [reply scripts/netobserv.sh:8]: Agreed - will make the change

**Review by memodi — APPROVED** [review]
/lgtm

**Pull request merged**